core/http/src/uri/uri.rs

```diff
@@ -316,3 +325,264 @@ macro_rules! impl_uri_from {
 impl_uri_from!(Origin);
 impl_uri_from!(Authority);
 impl_uri_from!(Absolute);
+
+mod uri_serde {
+    #![cfg(feature = "serde")]
+    
```

Comment:
```suggestion
#[cfg(feature = "serde")]
mod uri_serde {
```

Place the attr on the `mod`.